livekit-plugins/livekit-plugins-speechify/livekit/plugins/speechify/version.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2024 LiveKit, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+__version__ = "0.1.0"
```

Comment:
since plugin is for 1.x we'd need it to match agents version. there's a script that will update this as we release it

```suggestion
__version__ = "1.0.0"
```